### ISSUE TITLE
training starts with 500,000's iteration

### ISSUE THREAD

**Issue by hyunahOh** [AlexeyAB/darknet#3834]
I'm training with my own dataset and this is my config file.

```
\# Testing
\# batch=1
\# subdivisions=1
\# Training
batch=64
subdivisions=16
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 1000200
policy=steps
steps=400000,450000
scales=.1,.1
````

At first, max_batches was 500200, but the training stops right after starts.

I changed max_batches=1000200, and problem solved. (I can't understand why the first iteration starts at 500200)

Training with above condition is not problem, but my loss chart.png starts 5,000,200 and it doesn't look good.

> 
> #My Pretrained Model : yolov3.weights
> #Command : ./darknet detector train data/ecp.data data/yolov3-ecp.cfg yolov3.weights -dont_show -mjpeg_port 8888 -map -gpus 2
>

**Comment by kossolax** [comment]
You should start with this file: http://pjreddie.com/media/files/darknet53.conv.74 instead.

**Comment by hyunahOh** [comment]
> You should start with this file: http://pjreddie.com/media/files/darknet53.conv.74 instead.

Thank you! problem solved. But I wonder why it works..

**Comment by AlexeyAB** [comment]
> But I can't understand why the first iteration starts at 500200. (I've never trained that much iters.)
> 
> #My Pretrained Model : yolov3.weights
> #Command : ./darknet detector train data/ecp.data data/yolov3-ecp.cfg yolov3.weights -dont_show -mjpeg_port 8888 -map -gpus 2

Because `yolov3.weights` is trained for 500200 iterations.

**Comment by gmayday1997** [comment]
you can set  the  command  flag -clear 1
